spring-cloud-app-broker-core/src/main/java/org/springframework/cloud/appbroker/deployer/BackingServicesProvisionService.java

```diff
@@ -28,4 +29,15 @@
 
 	Flux<String> deleteServiceInstance(List<BackingService> backingServices);
 
+	/**
+	 * Create service keys
+	 * @return A flux of credentials associated with each created service key
+	 */
+	Flux<Map<String, Object>> createServiceKeys(List<BackingServiceKey> backingServiceKeys);
+
+	/**
+	 * Delete service keys
+	 * @return a Flux of service key names that were deleted
```

Comment:
Please, delete comment to be consistent with the other methods.

Reply:
Same as https://github.com/spring-cloud/spring-cloud-app-broker/pull/292/files#r392504267